src/Libraries/Microsoft.Extensions.TimeProvider.Testing/FakeTimeProvider.cs

```diff
@@ -14,13 +15,16 @@ namespace Microsoft.Extensions.Time.Testing;
 /// </summary>
 public class FakeTimeProvider : TimeProvider
 {
-    internal static readonly DateTimeOffset Epoch = new(2000, 1, 1, 0, 0, 0, 0, TimeSpan.Zero);
-
     internal readonly List<FakeTimeProviderTimer.Waiter> Waiters = new();
 
-    private DateTimeOffset _now = Epoch;
+    private DateTimeOffset _now;
     private TimeZoneInfo _localTimeZone;
 
+    /// <summary>
+    /// Gets the time at which the instance of the <see cref="FakeTimeProvider"/> was initialized.
+    /// </summary>
+    public DateTimeOffset StartTime { get; } = new DateTimeOffset(2000, 1, 1, 0, 0, 0, 0, TimeSpan.Zero);
```

Comment:
I am not sure what to do about the `error R9A049: (NETCORE_ENGINEERING_TELEMETRY=Build) Newly added symbol 'Microsoft.Extensions.Time.Testing.FakeTimeProvider.StartTime' must be marked as experimental (https://TODO/r9a049)`. 

Please advice.

Reply:
I think any new APIs must be marked as experimental, @rafal-mz and @geeknoid can provide mor clarity on this.

Reply:
Correct, since this is changing the API signature, it needs to be marked as [Experimental] until the API has been reviewed and approved.

Reply:
OK. How do I do that? An `[Experimental]` on the property?

Reply:
Ahh it seems a bunch of libraries have their own copy of [ExperimentalAttribute.cs](https://github.com/dotnet/extensions/blob/main/src/LegacySupport/ExperimentalAttribute/ExperimentalAttribute.cs). Do I just copy this file into this project and add the attribute?

Also, I am not sure why this is needed in this case. `FakeTimeProvider` has never been released before, so everything related to it is experimental, or?

Reply:
Correct. The attribute doesn't yet exist in the lower-levels of the runtime, but it exists in the src/LegacySupport folder. To get the definition, the .csproj file needs to have a <InjectExperimentalAttributeOnLegacy>true</InjectExperimentalAttributeOnLegacy>